be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -173,7 +173,11 @@ OLAPStatus BetaRowsetWriter::flush_single_memtable(MemTable* memtable, int64_t*
 RowsetSharedPtr BetaRowsetWriter::build() {
     // TODO(lingbin): move to more better place, or in a CreateBlockBatch?
     for (auto& wblock : _wblocks) {
```

Comment:
I think we should fix it in a more complete way, to return Status instead of nullptr, which is very error-prone.
And use `RowsetSharedPtr` as an out parameter.

Reply:
Now the caller handle the error according to whether the pointer is nullptr. So it‘s ok  to return nullptr, although it is not a complete way. There will be a lot of changes if return Status.